src/tools/gophish_complete.py

```diff
@@ -3,14 +3,15 @@
 """Complete a campaign in GoPhish and/or output a GoPhish campaign summary.
 
 Usage:
-  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--complete | --summary) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete CAMPAIGN_NAME (--complete | --summary) [--log-level=LEVEL] SERVER API_KEY
   gophish-complete (-h | --help)
   gophish-complete --version
 
 Options:
   API_KEY                   GoPhish API key.
+  CAMPAIGN_NAME             GoPhish campaign name.
   SERVER                    Full URL to GoPhish server.
-  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
   -c --complete             Display a list of campaigns to choose which to complete.
   -s --summary              Choose a campaign from a list to output a summary.
```

Comment:
The option descriptions for `--complete` and `--summary` should be updated to only mention their function- there should not be any mention about displaying a list of campaigns or choosing a campaign, since that is now going to depend on whether or not the optional campaign name has been provided.

Reply:
Fixed in commit 1ecf246.